samples/sampler/stories/default/TemperatureControl.js

```diff
@@ -18,6 +18,7 @@ storiesOf('Agate', module)
 		'TemperatureControl',
 		() => (
 			<TemperatureControl
+				disabled={boolean('disabled', Config)}
```

Comment:
Let's add this prop in `ArcSlider` story too.

Reply:
ok